### PR TITLE
Add tooltip to one more place in FC

### PR DESCRIPTION
**Pull request by NakedFool**
## What does this PR do and why?

Improvement on this https://github.com/jbx-protocol/juice-interface/issues/1944

- Added tooltip to one more place in FC
- Small improvements to existing code from the previous PR

## Screenshots or screen recordings

image
## Acceptance checklist

- [x] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [x] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [x] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

**Comment by tomquirk** [comment]
@NakedFool this looks pretty good, would you mind fixing the conflicts pls then handing to @johnnyd-eth for a final review? 🙏

**Comment by NakedFool** [comment]
> @NakedFool this looks pretty good, would you mind fixing the conflicts pls then handing to @johnnyd-eth for a final review? 🙏

yessir 🍻

File: src/components/Project/SpendingStats.tsx [hunk 1]
```diff
@@ -51,8 +51,8 @@ export default function SpendingStats({
       <div>
         <Tooltip
           title={
-            currency === 'ETH' ? (
-              <ETHToUSD ethAmount={distributableAmount ?? ''} />
+            currency === 'ETH' && distributableAmount?.gt(0) ? (
```
Comment (johnnyd-eth) [inline src/components/Project/SpendingStats.tsx:54]: **Nice**

**Comment by johnnyd-eth** [comment]
Thank you @NakedFool this looks all good to me!

**Review by johnnyd-eth — APPROVED** [review]

**Pull request merged**